### ISSUE TITLE
Support 9.5 and 9.4 in integration tests

### ISSUE THREAD

**Issue by x4m** [wal-g/wal-g#152]
As far as I can see (per reports) 0.2.0+ is broken for 9.5:
1. When we are doing backup from replica we check timeline by calling pg_checkpoint_control()
2. This function is 9.6+
3. Somehow we refactored master backup code that it obtains timeline, not from WAL filename, but the same way as in replica

But all these are not a big deal per se, will test and fix eventually.
We should:
1. either drop support of 9.4 and 9.5 entirely (recommending users to continue to use WAL-G from versions 0.1.* and REL0_1_STABLE)
2. Or add integration tests for every version, especially that are not part of hand QA checks

Thoughts?

**Comment by x4m** [comment]
actuall 9.5 works, just makes a warning

**Comment by johannesluedke** [comment]
@x4m is Postgres 9.4 supported by wal-g?

**Comment by x4m** [comment]
@johannesluedke Yes, but it is not tested with integration tests. If you observe some problems with PostgreSQL 9.4, please report them.
